aio/src/app/custom-elements/events/events.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit } from '@angular/core';
+
+import { EventsService } from './events.service';
+
+export interface Event {
+  endDate: string;      // "2020-05-27",
+  address: string;      // "Oslo, Norway",
+  title: string;        // "ngVikings",
+  linkUrl: string;      // "https://ngvikings.org/",
+  name: string;          // "ngVikings",
+  info: string;          // "May 25-26 conference, 27 workshops, 2020"
+}
+
+@Component({
+  selector: 'aio-events',
+  templateUrl: 'events.component.html'
+})
+export class EventComponent implements OnInit {
+
+  prevEvents: Event[];
```

Comment:
Nit: I think "past events" is more clear:

```suggestion
  pastEvents: Event[];
```